test/functional/apps/visualize/index.ts

```diff
@@ -21,101 +21,62 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
     before(async () => {
       log.debug('Starting visualize before method');
       await browser.setWindowSize(1280, 800);
+      await esArchiver.load('empty_kibana');
+
       await esArchiver.loadIfNeeded('logstash_functional');
       await esArchiver.loadIfNeeded('long_window_logstash');
-      await esArchiver.load('visualize');
-      await kibanaServer.uiSettings.replace({
-        defaultIndex: 'logstash-*',
-        [UI_SETTINGS.FORMAT_BYTES_DEFAULT_PATTERN]: '0,0.[000]b',
-      });
+
       isOss = await deployment.isOss();
     });
 
     // TODO: Remove when vislib is removed
-    describe('new charts library', function () {
+    describe('new charts library visualize ciGroup7', function () {
       this.tags('ciGroup7');
 
-      before(async () => {
-        await kibanaServer.uiSettings.update({
-          'visualization:visualize:legacyChartsLibrary': false,
-        });
-        await browser.refresh();
-      });
-
-      after(async () => {
-        await kibanaServer.uiSettings.update({
-          'visualization:visualize:legacyChartsLibrary': true,
-        });
-        await browser.refresh();
-      });
-
-      // Test replaced vislib chart types
-      loadTestFile(require.resolve('./_area_chart'));
-      loadTestFile(require.resolve('./_line_chart_split_series'));
-      loadTestFile(require.resolve('./_line_chart_split_chart'));
-      loadTestFile(require.resolve('./_point_series_options'));
-      loadTestFile(require.resolve('./_vertical_bar_chart'));
-      loadTestFile(require.resolve('./_vertical_bar_chart_nontimeindex'));
+      before(async () => await update(false));
```

Comment:
I don't see a good reason for further abstracting this code?

Reply:
Just dropping duplication, more `DRY`.